packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -474,6 +476,39 @@ export default class RouterService extends Service {
   */
 }
 
+if (EMBER_ROUTING_ROUTER_SERVICE_REFRESH) {
+  RouterService.reopen({
+    /**
+     * Refreshes all currently active routes, doing a full transition.
+     * If a route name is provided and refers to a currently active route,
+     * it will refresh only that route and its descendents.
+     * Returns a promise that will be resolved once the refresh is complete.
+     * All resetController, beforeModel, model, afterModel, redirect, and setupController
+     * hooks will be called again. You will get new data from the model hook.
+     *
+     * @method refresh
+     * @param {String} [pivotRouteName] the route to refresh (along with all child routes)
+     * @return Transition
+     * @public
+     */
```

Comment:
Can you add `@category` here?

Reply:
I'm not sure what that means exactly. Do you mean like [this](https://github.com/emberjs/ember.js/blob/b97e2019e86b3c203554b0b5d16107add90e8e34/packages/%40ember/component/template-only.ts#L28)?

It looks like the feature flag key, so `@category EMBER_ROUTING_ROUTER_REFRESH` in this case?

Reply:
Yes, exactly!